README.md

```diff
@@ -23,11 +31,10 @@ remove deployment platforms, or test with a different suite.
 * Automatic package version control with [Versioneer](https://github.com/warner/python-versioneer)
 * Sample data inclusion with packaging instructions
 * Basic documentation structure powered by [Sphinx](http://www.sphinx-doc.org/en/master/)
-* Automatic license file inclusion from several common Open Source licenses (optional)
 
 ## Requirements
 
-* Python 3.7, 3.8, or 3.9
+* Python 3.7+
```

Comment:
3.8+? (or is this a requirement for just running the cookiecutter)?

Reply:
Yeah, I can run it with 3.7 so didn't quite update this -- happy to change over to avoid confusion, though.

Reply:
I would suggest doing so, saves us from having to update it later too :)